hazelcast-client/src/main/java/com/hazelcast/client/proxy/ClientReplicatedMapProxy.java

```diff
@@ -93,7 +93,7 @@ public ClientReplicatedMapProxy(String serviceName, String objectName) {
     @Override
     protected void onInitialize() {
         int partitionCount = getContext().getPartitionService().getPartitionCount();
-        targetPartitionId = ThreadLocalRandom.current().nextInt(partitionCount);
+        targetPartitionId = ThreadLocalRandomProvider.get().nextInt(partitionCount);
```

Comment:
I just closed the PR which would have removed that :D https://github.com/hazelcast/hazelcast/pull/9690

Jaromir and I just discussed this with the following outcome: Since `UuidUtil` uses the same `Random` instance now, we might profit from keeping `ThreadLocalRandomProvider`, since the `Random` instance might already be there (or maybe re-used later by a UUID creation). Does this make sense to you or did we miss something?